src/viam/sdk/config/viam-cpp-sdk-libviamsdk.pc.in

```diff
@@ -1,12 +1,11 @@
-prefix=@CMAKE_INSTALL_PREFIX@
+prefix=/usr
 libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
-includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
+includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@/viam/sdk
```

Comment:
I'm not sure why this was changed. The `viam/api` tacked onto the include path for the `libviamapi.pc.in` is peculiar to the `viamapi` library, and is a side effect of some unfortunate naming issues in our `api` repository. But the `sdk` library doesn't have those problems, and the intended include root for the `sdk` library is definitely the `INSTALL_INCLUDEDIR`.